toolkit/components/uniffi-bindgen-gecko-js/src/templates/Scaffolding.cpp

```diff
@@ -6,6 +6,11 @@
 #include "mozilla/dom/{{ ci.scaffolding_class() }}.h"
 #include "mozilla/dom/OwnedRustBuffer.h"
 #include "mozilla/dom/Promise.h"
+#include "mozilla/Logging.h"
+#include "mozilla/EndianUtils.h"
+
+static mozilla::LazyLogModule sUniFFI{{ ci.scaffolding_class() }}Logger("uniffi_logger");
```

Comment:
This could be useful for debugging and was a life savior when I was getting `BAD_ACCESS` errors that were dumping the whole stack 🙈 

if we like it, I wouldn't mind sprinkling a few more logs around the code

You'd need to set the `MOZ_LOG` environment variable to include `uniffi_logger:<LEVEL>`, I use `uniffi_logger:3` so `INFO`s show up

Reply:
Seems like a great addition to me.